src/openrct2/object/RideObject.cpp

```diff
@@ -775,6 +781,22 @@ rct_ride_entry_vehicle RideObject::ReadJsonCar(const json_t* jCar)
     car.draw_order = ObjectJsonHelpers::GetInteger(jCar, "drawOrder");
     car.num_vertical_frames_override = ObjectJsonHelpers::GetInteger(jCar, "numVerticalFramesOverride");
 
+    auto jAnimationSpeed = json_object_get(jCar, "animationSpeed");
+    car.animation_speed_multiplier = ObjectJsonHelpers::GetInteger(
+        jAnimationSpeed, "multiplier", VehicleDefaultAnimationSpeeds[car.animation] >> 5);
+    car.animation_speed_exponent = ANIMATION_SPEED_MAX_EXPONENT
+        - ObjectJsonHelpers::GetInteger(
```

Comment:
This is now spanning four lines. For legibility, this should be broken up. `ANIMATION_SPEED_MAX_EXPONENT - (VehicleDefaultAnimationSpeeds[car.animation] & 0x1F)` can be saved in a separate constexpr, and the result of `ObjectJsonHelpers::GetInteger(...` in a separate variable as well.